packages/core/src/util/renderer.ts

```diff
@@ -98,6 +98,19 @@ export const computeLabel = (
   return required && !hideRequiredAsterisk ? label + '*' : label;
 };
 
+/**
+ * Decides whether to mark a field as required.
+ *
+ * @param {boolean} required whether the label belongs to a control which is required
+ * @returns {boolean} should the field be marked as required
+ */
+ export const displayRequired = (
```

Comment:
I would prefer renaming this a bit for more natural reading, i.e. `displayAsRequired`. Maybe also use `show` instead of `display`?
```suggestion
 export const showAsRequired = (
```